src/components/actions/Actions.jsx

```diff
@@ -0,0 +1,27 @@
+import { Fragment } from "react";
+import Button from "src/components/button/Button";
+import {
+  ALT_MARK_TEXT,
+  ALT_EDIT_TEXT,
+  ALT_DELETE_TEXT,
+} from "src/utils/constants/altTexts";
+import {
+  ICON_MARK,
+  ICON_PEN,
+  ICON_BIN,
+} from "src/utils/constants/imageSources";
+
+const Actions = () => {
+  const icons = [ICON_MARK, ICON_PEN, ICON_BIN];
+  const altTexts = [ALT_MARK_TEXT, ALT_EDIT_TEXT, ALT_DELETE_TEXT];
```

Comment:
Is there any reason behind maintaining two arrays for buttons ? Can we encapsulate the related properties of button in single object ?